### PR TITLE
Update transaction structure description.

### PR DESCRIPTION
**Pull request by Diiaablo95**
The description of the content of a transaction, and in particular a node transaction, is not consistent with the current workings of indy-node. While in the [transactions README](https://github.com/hyperledger/indy-node/blob/master/docs/source/transactions.md) is stated that the `dest` field of a node transaction is a node DID, it is instead the node verification key (as also shown by the test scripts, where the verification key of the node is written in the pool transaction, and not it public key).

### CODE REVIEW

**Comment by hyperledger-github** [comment]
Could one of the committers please verify this patch?

File: docs/source/transactions.md [hunk 1]
```diff
@@ -458,16 +458,10 @@ Adds a new node to the pool or updates an existing node in the pool
 
 - `dest` (base58-encoded string):
 
-    Target Node's DID as base58-encoded string for 16 or 32 byte DID value.
+    Target Node's verkey as base58-encoded string for 16 or 32 byte DID value.
```
Comment (ashcherbakov) [inline docs/source/transactions.md]: Yes,  you are right. abbreviated verkeys are currently not supported. Can you please also fix the documentation for NODE request in `requests.md`?

**Comment by ashcherbakov** [comment]
(ci) test this please

File: docs/source/transactions.md (hunk 1 above)
Comment (Diiaablo95) [reply docs/source/transactions.md]: Hey! Just updated the same info in both `requests.md` and `requests-new.md`.

**Comment by ashcherbakov** [comment]
(ci) test this please

File: docs/source/transactions.md (hunk 1 above)
Comment (ashcherbakov) [reply docs/source/transactions.md]: Thank you very much!

**Pull request merged**